conf/config-default.yaml

```diff
@@ -115,6 +115,8 @@ nginx_config:                     # config for render the template to genarate n
   worker_shutdown_timeout: 240s     # timeout for a graceful shutdown of worker processes
   event:
     worker_connections: 10620
+  envs:
+    - TEST
```

Comment:
We can add `APISIX_PROFILE` as the default.

Reply:
I changed my idea. We should always include `APISIX_PROFILE`.
Since the `TEST` is useless now, we should comment it out and just let it as an example.